src/components/ProductCard/ProductCard.tsx

```diff
@@ -23,7 +24,11 @@ const ProductCard = (props: ProductCardProps) => {
       </div>
       <h2 className='product-name'>{props.name}</h2>
       <h3 className='product-brand'>{props.brand}</h3>
-      <p className='product-tags'>tags: {props.tags}</p>
+
+      {props.tags.length > 0 ? (
```

Comment:
Can we do just `props.tags.length` without the greater than 0 here since 0 is already false and greater than 0 is already true?

Reply:
Dang I do not know why I did that lol it's fixed!